dashboard/http_server_head.py

```diff
@@ -119,19 +119,25 @@ async def metrics_middleware(self, request, handler):
         start_time = time.monotonic()
         try:
             response = await handler(request)
-            status_tag = f"{floor(response.status / 100)}xx"
+            try:
+                status_tag = f"{floor(response.status / 100)}xx"
+            except Exception:
+                status_tag = "unknown"
             return response
```

Comment:
In this case this can return None (per this issue https://github.com/ray-project/ray/issues/29632). What's the behavior of aiohttp when None is returned?